tests/ffi/class_in_ns.rs

```diff
@@ -0,0 +1,21 @@
+// To test receivers on a type in a namespace outide
+// the default. cxx::bridge blocks can only have a single
+// receiver type, and there can only be one such block per,
+// which is why this is outside.
```

Comment:
If I'm remembering correctly, I don't think this is the case currently, or if it is then we should fix it.

I would expect there to be able to be one "implicit" receiver type per `extern` block, multiple "explicit" receiver types per `extern` block, and multiple `extern` blocks per FFI module.

```rust
#[cxx::bridge]
mod ffi {
    extern "C" {
        type A;
        fn f(&self); // implicitly receiver A
    }
    extern "C" {
        type B;
        type C;
        fn f(self: &B); // explicit receivers
        fn f(self: &C);
    }
}
```

Reply:
This is exactly the thing to which I was alluding in https://github.com/dtolnay/cxx/issues/376#issuecomment-719072633 and I'm happy to say you're exactly right. Multiple explicit receivers do work. I will address this when I go through and make the cleanup PRs.